src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -125,6 +139,11 @@ private String extractStudentRegistrationKey(StudentAttributes student) {
         return joinLink.substring(startIndex);
     }
 
+    private boolean isRequestedFromPage(String requestorUrl) {
```

Comment:
I wouldn't call the argument `requestorUrl`, because this method is verifying if the request url contains a certain `Const.ActionURIs`. Perhaps something alongs the lines of `UrlToVerify` is more suitable. `requestorUrl` implies that it is the requesting url, which isn't what you are trying to say.